WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -125,6 +127,7 @@ final class HubMenuViewModel: ObservableObject {
          inboxEligibilityChecker: InboxEligibilityChecker = InboxEligibilityUseCase(),
          blazeEligibilityChecker: BlazeEligibilityCheckerProtocol = BlazeEligibilityChecker(),
```

Comment:
Should we inject `CIABEligibilityChecker` to this checker too?

```suggestion
         blazeEligibilityChecker: BlazeEligibilityCheckerProtocol? = nil,
```

then inside init:

```Swift
self.blazeEligibilityChecker = blazeEligibilityChecker ?? BlazeEligibilityChecker(siteCIABEligibilityChecker: siteCIABEligibilityChecker)
```

Reply:
Thx for the attention here. 
I'd still prefer to leave it. If we make a fallback to `BlazeEligibilityChecker(...)` for the nil `blazeEligibilityChecker` argument and them purposely (for whatever reason) pass `nil` - it will be neglected and overridden by the `??` operator. 
Currently the `BlazeEligibilityChecker()` initializer already has a default value for the `siteCIABEligibilityChecker` so we should be good. Even if there are different `CIABEligibilityChecker` instances  - they can work independently and show correct results.